client/src/visualiser-debugger/Component/CodeEditor/CodeEditor.tsx

```diff
@@ -6,12 +6,12 @@ import 'ace-builds/src-noconflict/theme-tomorrow';
 import { useState, useEffect } from 'react';
 import { useUserFsStateStore } from '../../Store/userFsStateStore';
 import { IFileFileNode } from '../FileTree/FS/IFileSystem';
+import { useFrontendStateStore } from '../../Store/frontendStateStore';
 
-type CodeEditorProps = {
-  currLine?: number;
-};
-
-const CodeEditor: React.FC<CodeEditorProps> = ({ currLine = 0 }: CodeEditorProps) => {
+const CodeEditor: React.FC = () => {
```

Comment:
I think React.FC is not needed anymore

https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/